### DISCUSSION TITLE
v1.15.0 not starting

### DISCUSSION THREAD

**Discussion by Siggemada** [brndnmtthws/conky#1286] [General]
What dependency do I need to install? I am on KDE Neon 20.04.

sudo ./conky-x86_64.AppImage
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by ./conky-x86_64.AppImage)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.13' not found (required by ./conky-x86_64.AppImage)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by ./conky-x86_64.AppImage)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by ./conky-x86_64.AppImage)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by ./conky-x86_64.AppImage)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libical.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmicrohttpd.so.12)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmicrohttpd.so.12)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libncurses.so.6)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libtinfo.so.6)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmysqlclient.so.21)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmysqlclient.so.21)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmysqlclient.so.21)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/liblua5.3.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libglib-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libglib-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libglib-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libImlib2.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libImlib2.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libsystemd.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libsystemd.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libsystemd.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpulse.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpulse.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libcurl-gnutls.so.4)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libcurl-gnutls.so.4)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libxml2.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libxml2.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libicuuc.so.70)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libicuuc.so.70)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgnutls.so.30)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgnutls.so.30)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libcrypto.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libcrypto.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libdbus-1.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libdbus-1.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgio-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgio-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/liblzma.so.5)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/liblzma.so.5)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libcap.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgcrypt.so.20)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpulsecommon-15.99.so)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpulsecommon-15.99.so)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpulsecommon-15.99.so)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libssh.so.4)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libpsl.so.5)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgssapi_krb5.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libldap-2.5.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libldap-2.5.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libunistring.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libunistring.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libgmodule-2.0.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmount.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libmount.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libselinux.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libselinux.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libsndfile.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libasyncns.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libasyncns.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libapparmor.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libapparmor.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libkrb5.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libkrb5.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libk5crypto.so.3)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libkrb5support.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libkrb5support.so.0)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libsasl2.so.2)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libblkid.so.1)
./conky-x86_64.AppImage: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/.mount_conky-QXfFBV/usr/bin/../lib/libFLAC.so.8)

**Comment by brndnmtthws** [comment]
You need to either install a newer version of glibc, or try compiling from source code.
